README.md

```diff
@@ -24,7 +24,7 @@ BinaryBuilder.run_wizard()
 
 4. The output of a build is a JLL package (typically hosted within the [JuliaBinaryWrappers](https://github.com/JuliaBinaryWrappers/) GitHub organization) which can be added to packages just like any other Julia package.  The JLL package will export bindings for all products defined within the build recipe.
 
-For more information, see the documentation for this package, viewable either directly in markdown within the [`docs/src`](docs/src) folder within this repository, or [online](https://juliapackaging.github.io/BinaryBuilder.jl/latest).
+For more information, see the documentation for this package, viewable either directly in markdown within the [`docs/src`](docs/src) folder within this repository, or [online](https://juliapackaging.github.io/BinaryBuilder.jl/dev).
```

Comment:
Maybe this one could point to `stable`?
```suggestion
For more information, see the documentation for this package, viewable either directly in markdown within the [`docs/src`](docs/src) folder within this repository, or [online](https://juliapackaging.github.io/BinaryBuilder.jl/stable).
```

Reply:
Sure! Makes more sense to me, too, but I wanted to avoid silently doing semantic changes. Should I amend the PR (and mention this in the commit message) or do you prefer to add an explicit separate commit?

Reply:
Separate commit is fine :+1:

Reply:
Could also leave it out, Documenter redirects nowadays

Reply:
@fredrikekre just to clarify, you suggest to  just put in there https://juliapackaging.github.io/BinaryBuilder.jl  yes?   BTW, that redirects to https://docs.binarybuilder.org/ -- should perhaps *that* URL be used everywhere?